src/sentry/tasks/statistical_detectors.py

```diff
@@ -220,8 +257,112 @@ def all_function_payloads(
         yield from function_payloads
 
 
-def query_transactions(project_id: int) -> None:
-    pass
+def query_transactions(
+    org_ids: List[int],
+    project_ids: List[int],
+    start: datetime,
+    end: datetime,
+    transactions_per_project: int,
+) -> List[DetectorPayload]:
+
+    duration_metric_id = indexer.resolve(
+        UseCaseID.TRANSACTIONS, org_ids[0], str(TransactionMRI.DURATION.value)
+    )
+    transaction_name_metric_id = indexer.resolve(
+        UseCaseID.TRANSACTIONS,
+        org_ids[0],
+        "transaction",
+    )
```

Comment:
I'm not familiar with the indexer but does it matter that you're only using the `org_ids[0]` for all the orgs in the query? Does it index differently per org?

Reply:
Good question, let me put a comment in